bucket/jamovi.json

```diff
@@ -0,0 +1,34 @@
+{
+    "version": "1.6.8.0",
+    "description": "Free and open statistical software to bridge the gap between researcher and statistician",
+    "homepage": "https://www.jamovi.org",
+    "license": {
+        "identifier": "AGPL3",
+        "url": "https://github.com/jamovi/jamovi/blob/master/LICENSE.md"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://www.jamovi.org/downloads/jamovi-1.6.8.0-win64.zip",
+            "hash": "09CED789F6FD336C37FC3A6DA4A93BA9CA7E3BB53C54B5692EE9AA0E0F1FECFB"
```

Comment:
lowercase

Reply:
Updated for style consistency. Also updated for stable version (i.e. 1.2.27.0).